sdk/keyvault/azure-security-keyvault-common/src/keyvault_pipeline.cpp

```diff
@@ -70,8 +70,7 @@ std::unique_ptr<Azure::Core::Http::RawResponse> _internal::KeyVaultPipeline::Sen
     case Azure::Core::Http::HttpStatusCode::NoContent:
       break;
     default:
-      throw Azure::Security::KeyVault::_internal::KeyVaultException::CreateException(
-          std::move(response));
+      throw Azure::Core::RequestFailedException(response);
```

Comment:
```suggestion
      throw Azure::Core::RequestFailedException(std::move(response));
```

Reply:
ditto for others

Reply:
nope, can't do that , once the pointer is moved it's deleted, that methods are either called several times, or they need the pointers in a few places.

Reply:
> nope, can't do that , once the pointer is moved it's deleted, that methods are either called several times, or they need the pointers in a few places.

Sorry, I didn't get it. Can you elaborate why we cannot do that?